backend/app.js

```diff
@@ -1,27 +1,86 @@
+const axios = require('axios');
 const config = require('config');
 const express = require('express');
 const log = require('npmlog');
 const morgan = require('morgan');
+const passport = require('passport');
+const session = require('express-session');
+const OidcStrategy = require('passport-openidconnect').Strategy;
 
 const utils = require('./components/utils');
+const authRouter = require('./routes/auth');
 const v1Router = require('./routes/v1');
 
+const apiRouter = express.Router();
+
 const app = express();
 app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+app.use(express.urlencoded({
+  extended: false
+}));
 
 app.use(morgan(config.get('server.morganFormat')));
 
+app.use(session({
+  secret: config.get('oidc.clientSecret'),
+  resave: false,
+  saveUninitialized: true
+}));
+app.use(passport.initialize());
+app.use(passport.session());
+
+// Logging Setup
 log.level = config.get('server.logLevel');
-log.addLevel('debug', 1500, { fg: 'green' });
+log.addLevel('debug', 1500, {
+  fg: 'green'
+});
 
 // Print out configuration settings in verbose startup
-log.verbose(utils.prettyStringify(config));
+log.verbose('Config', utils.prettyStringify(config));
+
+// Resolves OIDC Discovery values and returns an OIDC Strategy Config
+async function getOidcDiscovery() {
+  try {
+    const response = await axios.get(config.get('oidc.discovery'));
+
+    log.verbose(arguments.callee.name, `WebAde Token: ${utils.prettyStringify(response.data)}`);
```

Comment:
Not the webade token right?

Reply:
Good catch. Made it more abstract log now that the callee describes what and where it's coming from.